Assets/Scripts/IntroCutsceneController.cs

```diff
@@ -0,0 +1,77 @@
+﻿using UnityEngine;
+
+public class IntroCutsceneController : MonoBehaviour
+{
+    public AudioSource labChairSfx;
+    public AudioSource bemisVoice;
+    public AudioClip bemisDialogue2;
+    public AudioClip enterSimulationSfx;
+    public Animator simulationBoothLightingAnim;
+
+    public float spaceStationFlightSpeed = 10f;
+    
+    private static readonly int EnterSimulationBoothTrigger = Animator.StringToHash("EnterSimulationBooth");
+    private static readonly int StartSimulationTrigger = Animator.StringToHash("StartSimulation");
+    private static readonly int Rotation = Shader.PropertyToID("_Rotation");
+
+    private bool skyboxShouldRotate = true;
+
+    private void Awake()
+    {
+        Cursor.visible = false;
+        Cursor.lockState = CursorLockMode.Locked;
+    }
+
+    private void Update()
+    {
+        // press escape to skip cutscene
+        if (Input.GetKeyDown(KeyCode.Escape)) LoadFirstLevel();
```

Comment:
We might not wanna let them skip the cutscene with esc -> maybe have that as a public field, default it to escape but rewrite on Awake/Start to SPACE or something else if the platform is WebGL (look at the physics cam controller as an example for how to do this).

Reply:
switched to SPACE or RETURN keys